constants/constants.js

```diff
@@ -1,53 +1,52 @@
-// for crawlers
```

Comment:
Similar to the comment made on ./combine.js, would it be better to use `const` for the variables here?

Reply:
Refactored to export default variables when used as import 😀